.gitignore

```diff
@@ -59,7 +59,7 @@ dist/
 downloads/
 eggs/
 .eggs/
-lib/
+# lib/
```

Comment:
Why is that change needed here?

Reply:
Since the source files for this project are located in ./lib/ , this .gitignore was causing any changes to the JS files to be ignored for git purposes.  It seemed quite peculiar to me that a project would .gitignore it's own source folder.

Reply:
Oh that's right - I guess it would be better to just remove this line instead of commenting it